crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4028,7 +4012,7 @@ pub struct AdyenPayoutEligibilityRequest {
 #[serde(rename_all = "camelCase")]
 pub struct PayoutCardDetails {
     #[serde(rename = "type")]
-    _type: String,
+    payment_method_type: String,
     number: String,
     expiry_month: String,
     expiry_year: String,
```

Comment:
Why number, expiry_month & expiry_year were kept as String rather than Secret? While logging the connector request it will expose all those infos. Use secret<String> instead